praca/praca.tex

```diff
@@ -142,6 +142,53 @@ \section{Przypadki użycia}
   \item Zapisać wyniki przeprowadzanych analiz do pliku.
 \end{enumerate}
 
+\section{Architektura}
+\textbf{Mopnik} jest napisany w języku Java z użyciem narzędzia Apache
+Maven automatyzującego budowę programu.
+\subsubsection{Wykorzystywanie biblioteki}
+Implementacja opiera się na wykorzystaniu następujących bibliotek dla języka
+Java:
+\begin{itemize}
+\item Swing -- biblioteka graficzna służąca do stworzenia GUI aplikacji.
+\item JXMapViewer2 -- biblioteka zapewniająca swingowy JPanel renderujący
+  kafelki mapy (\url{https://github.com/msteiger/jxmapviewer2}).
+\item osm-parser -- biblioteka parsująca pliki .osm do obiektów w Javie
+  (\url{https://github.com/imintel/osm-parser}).
+\item opencsv -- biblioteka do parsowania plików CSV
+  (\url{http://opencsv.sourceforge.net/})
+\end{itemize}
+Oprócz zewnętrznych bibliotek, program korzysta też z będącej częścią tego
```

Comment:
Jakiego "tego"?

Reply:
"tego" w sensie naszego licencjatu...

Reply:
tzn. no imho to brzydko wygląda, ale jeśli sądzisz, że jest ok, to może zostać